src/disjoint_mut.rs

```diff
@@ -654,14 +654,35 @@ mod debug {
     use std::thread::ThreadId;
 
     #[derive(Debug)]
-    struct DisjointMutBounds {
+    pub(crate) struct DisjointMutBounds {
```

Comment:
Nit: does this need to be crate public?

Reply:
It needs to be accessed from the parent module, so I guess it could be `pub(super)` instead? Will change.